package.json

```diff
@@ -2,6 +2,8 @@
   "name": "layuplist",
   "version": "1.0.0",
   "repository": "https://github.com/layuplist/ll2",
-  "author": "Jai Smith, Adam McQuilkin",
-  "license": "MIT"
+  "author": "dplanner-dev",
+  "license": "MIT",
+  "workspaces": ["infra", "schema", "client"],
+  "private": true
```

Comment:
This isn't compatible with the license

Reply:
needs to be private to be used as a yarn workspace root.